jaeger-core/src/test/java/io/jaegertracing/JaegerSpanTest.java

```diff
@@ -444,6 +509,38 @@ public void testNoExpandExceptionLogs() {
     assertEquals(ex, logData.get(0).getFields().get(Fields.ERROR_OBJECT));
   }
 
+  @Test
+  public void testNoLoggingingStackTracesOnDebug() {
+    log.setEnabledLevels(Level.WARN);
+    JaegerSpan jaegerSpan = (JaegerSpan)tracer.buildSpan("foo").start();
```

Comment:
I just got a PR merged yesterday that makes this cast unnecessary. Could you rebase your PR and check it?